application/system_processor/SystemProcessor.py

```
@@ -0,0 +1,21 @@
+
+    def __init__(self, logFolder):
+        self.logFolder = logFolder
+        self.parseLogFiles()
+        self.logFiles = []
+
+    def parseLogFiles(self):
+        files = os.listdir(self.logFolder)
+
+        for file in files:
+            _path = os.path.join(self.logFolder, file)
+            cdlFile = Cdl(_path)
```

Ruff (0.8.2)

Local variable `cdlFile` is assigned to but never used

Remove assignment to unused variable `cdlFile`

(F841)